shuup/front/apps/registration/templates/shuup/registration/company_register.jinja

```diff
@@ -1,25 +1,9 @@
 {% extends "shuup/front/base.jinja" %}
 {% from "shuup/front/macros/general.jinja" import render_field %}
+{% from "shuup/front/macros/checkout.jinja" import render_address_form %}
```

Comment:
Good catch. I first intended to use the version imported from checkout macros, but later decided to bake my own version, since the imported one wasn't good for this case after all.

I'll remove the import.